Medium_difficulty/taxSalary.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
```

Comment:
Program logic is incorrect. Try to under stand the question properly.

Reply:
Okay, I will update code.

Reply:
I have resolved